google/cloud/spanner/samples/samples.cc

```diff
@@ -139,6 +139,48 @@ void DropDatabase(std::vector<std::string> const& argv) {
   (argv[0], argv[1], argv[2]);
 }
 
+void InsertData(std::vector<std::string> const& argv) {
+  if (argv.size() != 3) {
+    throw std::runtime_error(
+        "insert-data <project-id> <instance-id> <database-id>");
+  }
+
+  // TODO(#377) - cache the client across sample functions.
+  namespace gcs = google::cloud::spanner;
+  gcs::Client client(
+      gcs::MakeConnection(gcs::MakeDatabaseName(argv[0], argv[1], argv[2])));
+
+  //! [START spanner_insert_data]
+  namespace gcs = google::cloud::spanner;
```

Comment:
I didn't notice this before, but `gcs::Foo` looks to me like `Foo` within the `google::cloud::storage` namespace. I realize the 3-letter abbreviation works here too, but I wonder if it will be confusing.

Suggestion: Maybe we should consider using a `spanner` namespace alias in this case instead of "gcs"?

Thoughts?

Reply:
I was not liking `gcs::` for the same reasons, changed to `spanner::` in all the samples.